### ISSUE TITLE
Screenshot are half size for dialog

### ISSUE THREAD

**Issue by afaucogney** [jraska/Falcon#5]
I use SpoonFalcon to capture dialog from the material-dialog lib https://github.com/afollestad/material-dialogs 

When I capture, I get this:

![1461792397293_chargement_login_screen](https://cloud.githubusercontent.com/assets/6554201/14869022/1ad6e0d6-0cd1-11e6-9f59-b5868d63c6cc.png)

Nothing on the log, tried 2 times = same results

**Comment by jraska** [comment]
Hi,

This is the whole screenshot? Can you please tell me which device you are testing on and maybe a bit about the dialog you are showing? I will try to reproduce and find the cause.

**Comment by afaucogney** [comment]
Yes this is the whole screenshot. I did a short mistake. This is an Activity that is shown as a dialog, then from this activity-dialog, I open another dialog that keeps the same size on screenshot (as what I see, the screenshot height seems to be the activity-dialog background (white frame) height, but not vertical_center). Just an assumption !

Here is the layout:

```
<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="wrap_content"
    android:layout_gravity="center"
    android:layout_marginLeft="@dimen/marge_small"
    android:layout_marginRight="@dimen/marge_small"
    android:background="@drawable/dialog_activity"
    android:orientation="vertical"
    android:paddingLeft="@dimen/marge_small"
    android:paddingRight="@dimen/marge_small">

    <TextView
        android:id="@+id/tv_SelectedText"
        fontPath="fonts/Besom.ttf"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:layout_marginBottom="@dimen/marge"
        android:gravity="center"
        android:hint=""
        android:text="@string/dialog_activity_auth_title"
        android:textAllCaps="true"
        android:textColor="@color/identite"
        android:textSize="@dimen/popup_title_textSize" />

    <FrameLayout
        android:layout_width="match_parent"
        android:layout_height="@dimen/marge"
        android:layout_marginBottom="@dimen/marge_small"
        android:visibility="gone"
        android:background="@color/com_facebook_blue">

        <com.facebook.login.widget.LoginButton
            android:id="@+id/login_with_facebook"
            android:layout_width="fill_parent"
            android:layout_height="@dimen/marge"
            android:layout_gravity="center"
            android:paddingLeft="@dimen/marge_small"
            android:text="@string/login_with_facebook"
            android:textSize="@dimen/placeHolder_textSize" />

    </FrameLayout>

   <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="@dimen/marge"
        android:layout_marginBottom="@dimen/marge_small"
        android:background="@color/com_facebook_blue"
        android:orientation="horizontal">

        <ImageView
            android:layout_margin="@dimen/marge_small_border"
            android:layout_width="@dimen/marge_small"
            android:layout_height="@dimen/marge_small"
            android:scaleType="fitCenter"
            android:src="@drawable/com_facebook_button_icon" />

        <Button
            android:id="@+id/login_with_facebook_fake"
            android:layout_width="fill_parent"
            android:layout_height="@dimen/marge"
            android:layout_marginLeft="@dimen/marge_small"
            android:layout_marginRight="@dimen/marge_small"
            android:background="@color/transparent"
            android:text="@string/login_with_facebook"
            android:textAllCaps="false"
            android:textColor="@color/white"
            android:textSize="@dimen/placeHolder_textSize" />
    </LinearLayout>

    <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="@dimen/marge"
        android:layout_marginBottom="@dimen/marge_small"
        android:background="@color/interaction"
        android:orientation="horizontal">

        <ImageView
            android:layout_width="@dimen/marge"
            android:layout_height="@dimen/marge"
            android:scaleType="fitCenter"

            android:src="@mipmap/ic_launcher" />


        <Button
            android:id="@+id/login_with_password"
            android:layout_width="fill_parent"
            android:layout_height="@dimen/marge"
            android:layout_marginLeft="@dimen/marge_small"
            android:layout_marginRight="@dimen/marge_small"
            android:background="@color/transparent"
            android:text="@string/login_with_password"
            android:textAllCaps="false"
            android:textColor="@color/white"
            android:textSize="@dimen/placeHolder_textSize" />
    </LinearLayout>



    <LinearLayout
        android:layout_width="match_parent"
        android:layout_height="54dp"
        android:layout_marginTop="@dimen/marge_small"
        android:gravity="end"
        android:orientation="horizontal">


        <fr.millezimsolutions.app.millezimu.Component.PaperButton xmlns:widget="http://schemas.android.com/apk/res-auto"
            android:id="@+id/pb_neutral"
            android:layout_width="0dp"
            android:layout_height="54dp"
            android:layout_weight="1"
            android:visibility="invisible"
            widget:paper_color="@color/white_contrast"
            widget:paper_corner_radius="@dimen/view_corner_radius"
            widget:paper_font="fonts/Besom.ttf"
            widget:paper_ripple_color="@color/interaction_light"
            widget:paper_shadow_color="@color/white_contrast"
            widget:paper_text="@string/buttonCancel"
            widget:paper_text_color="@color/interaction_contraste"
            widget:paper_text_size="@dimen/button_textSize" />

        <fr.millezimsolutions.app.millezimu.Component.PaperButton xmlns:widget="http://schemas.android.com/apk/res-auto"
            android:id="@+id/pb_negative"
            android:layout_width="0dp"
            android:layout_height="54dp"
            android:layout_weight="1"
            widget:paper_color="@color/white_contrast"
            widget:paper_corner_radius="@dimen/view_corner_radius"
            widget:paper_font="fonts/Besom.ttf"
            widget:paper_ripple_color="@color/interaction_light"
            widget:paper_shadow_color="@color/white_contrast"
            widget:paper_text="@string/buttonCancel"
            widget:paper_text_color="@color/interaction"
            widget:paper_text_size="@dimen/button_textSize" />

        <fr.millezimsolutions.app.millezimu.Component.PaperButton xmlns:widget="http://schemas.android.com/apk/res-auto"
            android:id="@+id/pb_positive"
            android:layout_width="0dp"
            android:layout_height="54dp"
            android:layout_weight="1"
            android:visibility="invisible"
            widget:paper_color="@color/white_contrast"
            widget:paper_corner_radius="@dimen/view_corner_radius"
            widget:paper_font="fonts/Besom.ttf"
            widget:paper_ripple_color="@color/interaction_light"
            widget:paper_shadow_color="@color/white_contrast"
            widget:paper_text="@string/validate"
            widget:paper_text_color="@color/interaction"
            widget:paper_text_size="@dimen/button_textSize" />


    </LinearLayout>

</LinearLayout>
```

Here is a screenshot done from AS preview

![capture d ecran 2016-04-28 a 09 19 12](https://cloud.githubusercontent.com/assets/6554201/14878216/68d2c1b4-0d22-11e6-9547-fa0cb74eea92.png)

**Comment by jraska** [comment]
Hi thank you, I think you could find a bug, I will try to reproduce and I will try to fix it soon.

**Comment by jraska** [comment]
Sorry for taking it so long. It should be fixed with e5a13cd. I will close the issue when I release it.

**Comment by jraska** [comment]
Version 1.0.2 released. If you can, please check it helped.
